addons/website/views/snippets/s_numbers_charts.xml

```diff
@@ -31,19 +31,20 @@
                         </div>
                     </div>
                 </div>
-                <div class="o_grid_item o_cc o_cc2 g-col-lg-5 g-height-12 col-lg-5 rounded" style="grid-area: 1 / 8 / 13 / 13; --grid-item-padding-y: 80px; --grid-item-padding-x: 40px; z-index: 4;">
+                <div class="o_grid_item o_cc o_cc2 g-col-lg-5 g-height-12 col-lg-5 rounded" style="grid-area: 1 / 8 / 13 / 13; --grid-item-padding-y: 64px; --grid-item-padding-x: 40px; z-index: 4;">
                     <div class="s_chart" data-type="doughnut" data-legend-position="none" data-tooltip-display="false" data-stacked="false" data-border-width="2" data-data='{"labels":["First","Second"],"datasets":[{"label":"One","data":["25","75"],"backgroundColor":["o-color-3","o-color-2"],"borderColor":["",""]}]}' data-snippet="s_chart" data-name="Chart">
+                        <p><br/></p>
                         <canvas style="box-sizing: border-box;" width="456" height="228"/>
                     </div>
                     <!-- Placeholder chart, to be displayed in the preview modal -->
-                    <svg class="s_dialog_preview d-block mx-auto" width="450" height="230" viewBox="0 0 100 110" xmlns="http://www.w3.org/2000/svg">
+                    <svg class="s_dialog_preview d-block mt-3 mx-auto" width="450" height="230" viewBox="0 0 100 110" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This is an XML *class addition*, it can be ok but only if really required. If you are just improving the snippet from your POV, rather reserve this for master. You can imagine that a custo made on this snippet could be broken by you adding this class.